### PR TITLE
Adds a Syndicate Monkey Agent beacon uplink item

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

Adds a Syndicate Monkey Agent beacon uplink item. It spawns a dapper monkey that must follow your orders.

Added a monkey gun case to the uplink, which contains monkey guns! Though they aren't very powerful.

Added a more modularlike subtype for antagonist spawners to reduce future hardcoding.

Gave the syndicate turtleneck a monkey sprite, from SS14!

## Why It's Good For The Game

I want to see the clown driving security insane with 2-3 monkeys and an incredible amount of pranking. Or an assistant killing everyone with his monkey friends while wearing a monkey suit. Or a geneticist sending out mutated monkeys to kill people. Or a scientist equipping his monkeys with bombs or xenobiology equipment and sending them out to wreak havoc.

6 TC is only enough for two monkeys, but you can get a third if you finish any kind of objective.

> Added a monkey gun case to the uplink, which contains monkey guns! Though they aren't very powerful.

We can't have the monkey mafia without guns, come on. The guns are weak and only usable by monkeys. Additionally, they're restricted to entertainment only.

Credit to SS14 for the monky turtleneck sprite.

## Changelog

:cl:
add: Adds a Syndicate Monkey Agent beacon uplink item. It spawns a dapper monkey that must follow your orders.
add: Added a monkey gun case to the uplink, which contains monkey guns! Though they aren't very powerful.
refactor: Added a more modularlike subtype for antagonist spawners to reduce future hardcoding.
sprite: Gave the syndicate turtleneck a monkey sprite, from SS14!
/:cl:

### CODE REVIEW

**Comment by TheNobodyofimportance** [comment]
Amazing pr! I can already imagine the amount of carnage and mass honking that comes from an army of vent crawling monkes souped up with whatever job items you grinded during the shift. 

I do have a question though about this paragraph:
> 7 TC is only enough for two monkeys, but you can get a third if you finish any kind of objective.
Wdym by that? Does a single beacon give you multiple monkeys? You can't spend 7 tc on two of the same item.

**Comment by BlueMemesauce** [comment]
Is it impossible to humanize the monkeys? What happens if you give them mutadone?

**Comment by carlarctg** [comment]
> Is it impossible to humanize the monkeys? What happens if you give them mutadone?

The clever and monky mutations are mutadone locked.

**Comment by D4C-420** [comment]
> The clever and monky mutations are mutadone locked.

>I'm sure there's some highly elaborate and over complicated method, most likely involving xenobiology, that will be featured in a Livrah video in two months. But that's a feature.

You can directly edit genes with the DNA console. Humanizing them would just be walking to gene and throwing them into a scanner. Or just taking out the brain and throwing it into a new human body. 

In other words for 7 tc and 5-10 minutes of effort you get your very own blood brother. 

Eh I think its fine, about the same amount of effort as a hypnoflash.

**Comment by CRITAWAKETS** [comment]
clowns boutta unleash the trunk monkey on us

**Comment by Loafin34** [comment]
i like this but make sure to have a nuke ops one with a syndicate gorilla

**Comment by Wallemations** [comment]
> > Is it impossible to humanize the monkeys? What happens if you give them mutadone?
> 
> 
> 
> The clever and monky mutations are mutadone locked. 
> 
> 
> 
> I'm sure there's *some* highly elaborate and over complicated method, most likely involving xenobiology, that will be featured in a Livrah video in two months. But that's a feature.

Since, from what I can tell, the sentience is just an innate feature of them, I have a feeling it will be as simple as a brain surgery.
If they wanted to not have that happen they could have made the monkey's sentience use one of those monkey helmets

Then again this could have been dealt with already and I could be stupid lmao

**Comment by Yobrocharlie** [comment]
One potential problem with this would be if this is available to nuclear oeperatives because a monkey crawling through the vents with a power fist is very scary for anybody

**Comment by SomeoneYouProbablyKnow** [comment]
Don't nuckies have access to monkey cubes already (that + sentience potion)? They got a bunch of stuff in their base but maybe not cubes.

**Comment by carlarctg** [comment]
`#define MONKEY_AGENT_NAMES list("Al Chimpone", "Agent 9", "Agent Banana", "Agent Potassium", "Agent Ape","Aldo", "Banana Bond", "Bonobo Assassin", "Caesar", "Solid Simian", "Tony Bananas", "Murderous George", "Monkey Business", "Hit-Monkey")
`

More funny chimp agent names would be appreciated

File: code/modules/uplink/uplink_items/job.dm [hunk 1]
```diff
@@ -333,3 +333,25 @@
...
+	refundable = TRUE
+
+/datum/uplink_item/role_restricted/monkey_supplies
+	name = "Simian Agent Supplies"
+	desc = "Sometimes you need a bit more firepower than a rabid monkey. Such as a rabid, armed monkey! \
+		Monkeys can unpack this kit to recieve a bag with a bargain-bin gun, ammunition, and some miscellaneous supplies."
+	item = /obj/item/storage/toolbox/guncase/monkeycase
+	cost = 4
+	limited_stock = 3
+	restricted_roles = list(JOB_RESEARCH_DIRECTOR, JOB_SCIENTIST, JOB_GENETICIST, JOB_ASSISTANT, JOB_MIME, JOB_CLOWN)
```
Comment (carlarctg) [inline code/modules/uplink/uplink_items/job.dm:355]:
```suggestion
	restricted_roles = list(JOB_ASSISTANT, JOB_MIME, JOB_CLOWN)
```

**Comment by carlarctg** [comment]
bruh

**Comment by YakumoChen** [comment]
> The clever and monky mutations are mutadone locked.

Is this just a thing for syndicate monkies or all of them? Monkey humans are immensely useful for medbay and making it hard to make them would be very unfortunate 

Not to mention it really scuffs my ability as chef to source cannibal meat.

File: code/game/objects/items/storage/toolbox.dm [hunk 2]
```diff
@@ -447,3 +447,51 @@
...
+/obj/item/storage/toolbox/guncase/monkeycase/attack_hand(mob/user, list/modifiers)
+	if(!monkey_check(user))
+		return
+	. = ..()
+
+/obj/item/storage/toolbox/guncase/monkeycase/proc/monkey_check(mob/user)
+	if(ismonkey(user) || isgorilla(user))
+		return TRUE
+	to_chat(user, span_warning("You put your hand on the hand scanner, and it rejects it with an angry chimpanzee screech!"))
+	playsound(src, "sound/creatures/monkey/monkey_screech_[1-7].ogg", 40, TRUE)
```
Comment (MrMelbert) [inline code/game/objects/items/storage/toolbox.dm:469]: `1 - 7`?

Comment (carlarctg) [reply code/game/objects/items/storage/toolbox.dm:469]: my brain is enormos

File: code/__DEFINES/traits.dm [hunk 3]
```diff
@@ -282,6 +282,8 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_NO_ZOMBIFY "no_zombify"
 /// Carbons with this trait can't have their DNA copied by diseases nor changelings
 #define TRAIT_NO_DNA_COPY "no_dna_copy"
+/// Carbones with this trait cant have their dna scrambled by genetics or a disease retrovirus.
```
Comment (ATH1909) [inline code/__DEFINES/traits.dm:285]:
```suggestion
/// Carbons with this trait cant have their dna scrambled by genetics or a disease retrovirus.
```

File: code/modules/projectiles/pins.dm [hunk 4]
```diff
@@ -373,6 +373,16 @@
 	suit_requirement = /obj/item/clothing/suit/bluetag
 	tagcolor = "blue"
 
+/obj/item/firing_pin/monkey
+	name = "monkeylock firing pin"
+	desc = "This firing pin prevents non-monkies from firing a gun."
```
Comment (ATH1909) [inline code/modules/projectiles/pins.dm:378]:
```suggestion
	desc = "This firing pin prevents non-monkeys from firing a gun."
```

Comment (carlarctg) [reply code/modules/projectiles/pins.dm:378]: you hate soul

File: code/modules/projectiles/pins.dm [hunk 5]
```diff
@@ -373,6 +373,16 @@
...
+	fail_message = "not a monkey!"
+
+/obj/item/firing_pin/monkey/pin_auth(mob/living/user)
+	if(!ismonkey(user))
```
Comment (carlarctg) [inline code/modules/projectiles/pins.dm:382]:
```suggestion
	if(!is_simian(user))
```

File: code/modules/uplink/uplink_items/job.dm [hunk 6]
```diff
@@ -333,3 +333,25 @@
...
+/datum/uplink_item/role_restricted/monkey_supplies
+	name = "Simian Agent Supplies"
+	desc = "Sometimes you need a bit more firepower than a rabid monkey. Such as a rabid, armed monkey! \
+		Monkeys can unpack this kit to recieve a bag with a bargain-bin gun, ammunition, and some miscellaneous supplies."
+	item = /obj/item/storage/toolbox/guncase/monkeycase
+	cost = 4
+	limited_stock = 3
+	restricted_roles = list(JOB_ASSISTANT, JOB_MIME, JOB_CLOWN)
+	restricted = TRUE
+	refundable = TRUE
```
Comment (carlarctg) [inline code/modules/uplink/uplink_items/job.dm:357]:
```suggestion
	refundable = FALSE
```

File: code/__DEFINES/mobs.dm [hunk 7]
```diff
@@ -119,6 +119,7 @@
 #define SPECIES_LIZARD_SILVER "silverscale"
 #define SPECIES_NIGHTMARE "nightmare"
 #define SPECIES_MONKEY "monkey"
+#define SPECIES_MONKEY_NO_MUT "monkey_no_mutation"
```
Comment (Ghommie) [inline code/__DEFINES/mobs.dm:122]: Unused define, remove it.

**Review by Ghommie — CHANGES_REQUESTED** [review]
Nice, you might want to make the syndie (soft) space suit compatible with monkeys next. Space walks to and fro' the station and the Infiltrator might be a bit tricky otherwise.

**Comment by mc-oofert** [comment]
this would be way better off as a chameleon syndicate mind magnification helmet than shitcoding genetics

File: code/game/objects/items/storage/toolbox.dm [hunk 8]
```diff
@@ -447,3 +447,65 @@
...
+	desc = "Everything a monkey needs to truly go ape-shit. There's a paw-shaped hand scanner lock on the front of the case."
+
+/obj/item/storage/toolbox/guncase/monkeycase/Initialize(mapload)
+	. = ..()
+	atom_storage.locked = STORAGE_SOFT_LOCKED
+
+/obj/item/storage/toolbox/guncase/monkeycase/attack_self(mob/user, modifiers)
+	if(!monkey_check(user))
+		return
+	. = ..()
```
Comment (Jacquerel) [inline code/game/objects/items/storage/toolbox.dm:462]:
```suggestion
	return ..()
```

File: code/game/objects/items/boxcutter.dm [hunk 9]
```diff
@@ -17,6 +17,7 @@
 	var/snap_time_weak_handcuffs = 0 SECONDS
 	/// Used on Initialize, how much time to cut real handcuffs. Null means it can't.
 	var/snap_time_strong_handcuffs = null
+	var/start_extended = FALSE
```
Comment (Jacquerel) [inline code/game/objects/items/boxcutter.dm:20]: Document this, even if it is obvious

File: code/game/objects/items/storage/toolbox.dm [hunk 10]
```diff
@@ -447,3 +447,65 @@
...
+
+/obj/item/storage/toolbox/guncase/monkeycase/attack_self_secondary(mob/user, modifiers)
+	attack_self(user, modifiers)
+	return
+
+/obj/item/storage/toolbox/guncase/monkeycase/attack_hand(mob/user, list/modifiers)
+	if(!monkey_check(user))
+		return
+	. = ..()
```
Comment (Jacquerel) [inline code/game/objects/items/storage/toolbox.dm:471]:
```suggestion
	return ..()
```

File: code/game/objects/items/storage/toolbox.dm [hunk 11]
```diff
@@ -447,3 +447,65 @@
...
+		atom_storage.locked = STORAGE_NOT_LOCKED
+		to_chat(user, span_notice("You place your paw on the paw scanner, and hear a soft click as [src] unlocks!"))
+		playsound(src, 'sound/items/click.ogg', 25, TRUE)
+		return TRUE
+	to_chat(user, span_warning("You put your hand on the hand scanner, and it rejects it with an angry chimpanzee screech!"))
+	playsound(src, "sound/creatures/monkey/monkey_screech_[rand(1,7)].ogg", 75, TRUE)
+	return FALSE
+
+/obj/item/storage/toolbox/guncase/monkeycase/PopulateContents()
+	switch(rand(1, 100))
```
Comment (Jacquerel) [inline code/game/objects/items/storage/toolbox.dm:487]: why are you doing a 1-100 randomisation when you only have three equally likely outcomes?
It doesn't really _matter_ but it's a slightly confusing approach when `rand(1, 3)` would do 😅

File: code/modules/antagonists/_common/antag_spawner.dm [hunk 12]
```diff
@@ -283,3 +283,139 @@
...
+	/// Style used by the droppod
+	var/pod_style = STYLE_SYNDICATE
+	/// Do we use a random subtype of the outfit?
+	var/use_subtypes = TRUE
+	/// The antag role we check if the ghosts have enabled to get the poll.
+	var/poll_role_check = ROLE_TRAITOR
+	/// The mind's special role.
+	var/role_to_play = ROLE_SYNDICATE_MONKEY
+	var/poll_ignore_category = POLL_IGNORE_SYNDICATE
+	var/fail_text = "Unable to connect to Syndicate command. Please wait and try again later or use the beacon on your uplink to get your points refunded."
```
Comment (Jacquerel) [inline code/modules/antagonists/_common/antag_spawner.dm:313]: Missed some vars with your documentation

File: code/modules/antagonists/_common/antag_spawner.dm [hunk 13]
```diff
@@ -283,3 +283,139 @@
...
+	if(LAZYLEN(baddie_candidates))
+		if(QDELETED(src) || !check_usability(user))
+			return
+		used = TRUE
+		var/mob/dead/observer/G = pick(baddie_candidates)
+		spawn_antag(G.client, get_turf(src), user)
+		do_sparks(4, TRUE, src)
+		qdel(src)
+	else
+		to_chat(user, span_warning(fail_text))
```
Comment (Jacquerel) [inline code/modules/antagonists/_common/antag_spawner.dm:343]:
```suggestion
	if(!LAZYLEN(baddie_candidates))
		to_chat(user, span_warning(fail_text))
		return
	if(QDELETED(src) || !check_usability(user))
		return
	used = TRUE
	var/mob/dead/observer/ghostie = pick(baddie_candidates)
	spawn_antag(ghostie.client, get_turf(src), user)
	do_sparks(4, TRUE, src)
	qdel(src)
```

Early return & avoid single letter var names.

File: code/modules/antagonists/syndicate_monkey/syndicate_monkey.dm [hunk 14]
```diff
@@ -0,0 +1,36 @@
...
+	show_name_in_check_antagonists = TRUE
+	count_against_dynamic_roll_chance = FALSE
+	show_to_ghosts = TRUE
+	var/mob/living/monky_master
+
+/datum/antagonist/syndicate_monkey/on_gain()
+	monky_master = owner.enslaved_to?.resolve()
+	if(monky_master)
+		forge_objectives(monky_master)
+	. = ..()
```
Comment (Jacquerel) [inline code/modules/antagonists/syndicate_monkey/syndicate_monkey.dm:15]:
```suggestion
	return ..()
```

File: code/modules/antagonists/syndicate_monkey/syndicate_monkey.dm [hunk 15]
```diff
@@ -0,0 +1,36 @@
...
+
+/datum/antagonist/syndicate_monkey/Destroy()
+	monky_master = null
+	. = ..()
```
Comment (Jacquerel) [inline code/modules/antagonists/syndicate_monkey/syndicate_monkey.dm:19]:
```suggestion
	return ..()
```

File: code/modules/antagonists/syndicate_monkey/syndicate_monkey.dm [hunk 16]
```diff
@@ -0,0 +1,36 @@
+/datum/antagonist/syndicate_monkey
+	name = "\improper Syndicate Monkey"
+	antagpanel_category = ANTAG_GROUP_SYNDICATE
+	show_in_roundend = TRUE
+	show_in_antagpanel = TRUE
+	show_name_in_check_antagonists = TRUE
+	count_against_dynamic_roll_chance = FALSE
+	show_to_ghosts = TRUE
+	var/mob/living/monky_master
```
Comment (Jacquerel) [inline code/modules/antagonists/syndicate_monkey/syndicate_monkey.dm:9]: Document this please

File: code/modules/projectiles/guns/ballistic/automatic.dm [hunk 17]
```diff
@@ -218,10 +228,23 @@
 	bolt_type = BOLT_TYPE_OPEN
 	empty_indicator = TRUE
 	show_bolt_icon = FALSE
+	var/rof = 0.1 SECONDS
```
Comment (Jacquerel) [inline code/modules/projectiles/guns/ballistic/automatic.dm:231]: Let's just assume I also marked "pls document " on any other new vars.

File: code/modules/projectiles/guns/ballistic/automatic.dm [hunk 18]
```diff
@@ -218,10 +228,23 @@
...
+
+/obj/item/gun/ballistic/automatic/tommygun/chimpgun
+	name = "\improper Typewriter"
+	desc = "It was the best of times, it was the BLURST of times!? You stupid monkeys!"
+	fire_delay = 2
+	rof = 0.2 SECONDS
+	projectile_damage_multiplier = 0.4
+	projectile_wound_bonus = -25
+	pin = /obj/item/firing_pin/monkey
+	// Roughly 9 damage per bullet every 0.2 seconds, equaling out to downing an opponent in a bit over a second, if they have no armor.
```
Comment (Jacquerel) [inline code/modules/projectiles/guns/ballistic/automatic.dm:247]:
```suggestion
/**
 * Weak tommygun for syndicate chimps. It comes in a 4 TC kit.
 * Roughly 9 damage per bullet every 0.2 seconds, equaling out to downing an opponent in a bit over a second, if they have no armor.
 */
/obj/item/gun/ballistic/automatic/tommygun/chimpgun
	name = "\improper Typewriter"
	desc = "It was the best of times, it was the BLURST of times!? You stupid monkeys!"
	fire_delay = 2
	rof = 0.2 SECONDS
	projectile_damage_multiplier = 0.4
	projectile_wound_bonus = -25
	pin = /obj/item/firing_pin/monkey
```
Might as well just do this for all of these, it's a little weird to have comments that come after the type definition?

File: code/modules/projectiles/pins.dm (hunk 5 above)
Comment (Jacquerel) [reply code/modules/projectiles/pins.dm:382]: Bit pointless as I believe Gorillas are inherently unable to fire guns.

**Review by Jacquerel — CHANGES_REQUESTED** [review]
Mostly nitpicks.

File: code/modules/projectiles/pins.dm (hunk 5 above)
Comment (carlarctg) [reply code/modules/projectiles/pins.dm:382]: thats cringe

File: code/game/objects/items/storage/toolbox.dm (hunk 11 above)
Comment (carlarctg) [reply code/game/objects/items/storage/toolbox.dm:487]: oooga boooga

**Comment by carlarctg** [comment]
> Nice, you might want to make the syndie (soft) space suit compatible with monkeys next. Space walks to and fro' the station and the Infiltrator might be a bit tricky otherwise.

I looked into this but realized monkey variation is only supported for uniforms and I didn't want to refactor that shit. Monkeys can just do funni spacewalks,

**Comment by SomeoneYouProbablyKnow** [comment]
You *can* make a space proof uniform that only fits monkeys but that sounds like an equal pain in the ass

File: code/_globalvars/lists/names.dm [hunk 19]
```diff
@@ -24,6 +24,7 @@ GLOBAL_LIST_INIT(nightmare_names, world.file2list("strings/names/nightmare.txt")
 GLOBAL_LIST_INIT(megacarp_first_names, world.file2list("strings/names/megacarp1.txt"))
 GLOBAL_LIST_INIT(megacarp_last_names, world.file2list("strings/names/megacarp2.txt"))
 GLOBAL_LIST_INIT(cyberauth_names, world.file2list("strings/names/cyberauth.txt"))
+GLOBAL_LIST_INIT(syndicate_monkey_names, world.file2list("strings/names/syndicaet_monkey.txt"))
```
Comment (mc-oofert) [inline code/_globalvars/lists/names.dm:27]: that doesnt look spelled properly

Comment (carlarctg) [reply code/_globalvars/lists/names.dm:27]: look again dumbas

Comment (carlarctg) [reply code/_globalvars/lists/names.dm:27]:
```suggestion
GLOBAL_LIST_INIT(syndicate_monkey_names, world.file2list("strings/names/syndicate_monkey.txt"))
```

**Comment by ju45he** [comment]
Not reading 20 peoples comments but will tell you mine, this is going to be spammed and every round will have 17 ghosts waiting for the monkey drop in order for them to get nearly guranteed antag spawns so they can speedrunmaxcappingsecurityany% and instantly green text.

File: code/modules/projectiles/pins.dm [hunk 20]
```diff
@@ -373,6 +373,16 @@
...
 	tagcolor = "blue"
 
+/obj/item/firing_pin/monkey
+	name = "monkeylock firing pin"
+	desc = "This firing pin prevents non-monkeys from firing a gun."
+	fail_message = "not a monkey!"
+
+/obj/item/firing_pin/monkey/pin_auth(mob/living/user)
+	if(!is_simian(user))
+		return FALSE
```
Comment (carlarctg) [inline code/modules/projectiles/pins.dm:383]:
```suggestion
		playsound(get_turf(src), "sound/creatures/monkey/monkey_screech_[rand(1,7)].ogg", 75, TRUE)
		return FALSE
```

File: strings/names/syndicate_monkey.txt [hunk 21]
```diff
@@ -0,0 +1,16 @@
...
+Banana Bond
+Bonobo Assassin
+Caesar
+Evil Monkey
+Solid Simian
+Tony Bananas
+Koba
+Murderous George
+Monkey Business
+Hit-Monkey
```
Comment (carlarctg) [inline strings/names/syndicate_monkey.txt:16]:
```suggestion
Hit-Monkey
Guenter
Goku
Kill Julien
```

**Comment by SomeoneYouProbablyKnow** [comment]
> Not reading 20 peoples comments but will tell you mine, this is going to be spammed and every round will have 17 ghosts waiting for the monkey drop in order for them to get nearly guranteed antag spawns so they can speedrunmaxcappingsecurityany% and instantly green text.

The Good Ending

**Comment by carlarctg** [comment]
> Not reading 20 peoples comments but will tell you mine, this is going to be spammed and every round will have 17 ghosts waiting for the monkey drop in order for them to get nearly guranteed antag spawns so they can speedrunmaxcappingsecurityany% and instantly green text.

Sounds good to me chief

File: code/modules/projectiles/guns/ballistic/automatic.dm [hunk 22]
```diff
@@ -146,6 +146,16 @@
...
 
+// Weak uzi for syndicate chimps. It comes in a 4 TC kit.
+
+/obj/item/gun/ballistic/automatic/mini_uzi/chimpgun
+	name = "\improper MONK-10"
+	desc = "Developed by Syndicate monkeys, for syndicate Monkeys. Despite the name, this weapon resembles an Uzi significantly more than a MAC-10. Uses 9mm rounds. There's a label on the other side of the gun that says \"Do what comes natural.\""
+	projectile_damage_multiplier = 0.4
+	projectile_wound_bonus = -25
+	pin = /obj/item/firing_pin/monkey
+	// Roughly 9 damage per bullet every 0.2 seconds, equaling out to downing an opponent in a bit over a second, if they have no armor.
```
Comment (Jacquerel) [inline code/modules/projectiles/guns/ballistic/automatic.dm:157]:
```suggestion
/**
 * Weak uzi for syndicate chimps. It comes in a 4 TC kit.
 * Roughly 9 damage per bullet every 0.2 seconds, equaling out to downing an opponent in a bit over a second, if they have no armor.
 */
/obj/item/gun/ballistic/automatic/mini_uzi/chimpgun
	name = "\improper MONK-10"
	desc = "Developed by Syndicate monkeys, for syndicate Monkeys. Despite the name, this weapon resembles an Uzi significantly more than a MAC-10. Uses 9mm rounds. There's a label on the other side of the gun that says \"Do what comes natural.\""
	projectile_damage_multiplier = 0.4
	projectile_wound_bonus = -25
	pin = /obj/item/firing_pin/monkey
```

File: code/modules/projectiles/guns/ballistic/pistol.dm [hunk 23]
```diff
@@ -55,6 +55,17 @@
...
+// Weak 1911 for syndicate chimps. It comes in a 4 TC kit.
+
+/obj/item/gun/ballistic/automatic/pistol/m1911/chimpgun
+	name = "\improper CH1M911"
+	desc = "For the monkey mafioso on-the-go. Uses .45 rounds and has the distinct smell of bananas."
+	projectile_damage_multiplier = 0.5
+	projectile_wound_bonus = -12
+	pin = /obj/item/firing_pin/monkey
+
+// 15 damage every.. second? 7 shots to kill. Not fast.
```
Comment (Jacquerel) [inline code/modules/projectiles/guns/ballistic/pistol.dm:67]:
```suggestion
/**
 * Weak 1911 for syndicate chimps. It comes in a 4 TC kit.
 * 15 damage every.. second? 7 shots to kill. Not fast.
 */
/obj/item/gun/ballistic/automatic/pistol/m1911/chimpgun
	name = "\improper CH1M911"
	desc = "For the monkey mafioso on-the-go. Uses .45 rounds and has the distinct smell of bananas."
	projectile_damage_multiplier = 0.5
	projectile_wound_bonus = -12
	pin = /obj/item/firing_pin/monkey

```

**Review by Jacquerel — CHANGES_REQUESTED** [review]

**Pull request merged**